Formula/gusto-lima.rb

```diff
@@ -0,0 +1,32 @@
+class GustoLima < Formula
+  desc "Gusto's opinionated colima profile"
```

Comment:
This description leaves me wondering what is so opinionated about it 😁 

A cursory look seems to be more about installing a working set of packages, ie `docker` isn't an explicit dependency of `colima`.